CONTRIBUTING.md

```diff
@@ -54,6 +54,9 @@ we recommend the following workflow:
    tox -e fulltest-py312
```

Comment:
For checking if my code messed up sth I always used `pytest --fast` - or should tox be used? The former was useful as it was quick compared to `pytest` for smaller commits.

Reply:
you can use `pytest --fast` or tox via posargs a la `tox -e fulltest-py312 -- --force` (anything after the blank `--` will be transferred to the tox internal pytest call)
The fast tests are kind of explained in `tests/README.md`